index.rst

```diff
@@ -313,6 +313,7 @@ Environmental
     RuuviTag, components/sensor/ruuvitag, ruuvitag.jpg, Temperature & Humidity & Accelerometer
     SCD30, components/sensor/scd30, scd30.jpg, CO2 & Temperature & Humidity
     SCD4X, components/sensor/scd4x, scd4x.jpg, CO2 & Temperature & Humidity
+    ENS160, components/sensor/ens160, ens160.jpg, CO2 & Air Quality
```

Comment:
Please place this alphabetically within the category.
It should also be added into the "Air Quality" category above because it can provide this information, as well.